src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
```

Comment:
<img width="1055" alt="image" src="https://github.com/mate-academy/react_person/assets/86789161/757530fd-60f6-44ae-9698-7ba5b66b44e4">

Reply:
Use destructuring for all fields